src/contrib/cppheaderparser/CppHeaderParser/CppHeaderParser.py

```diff
@@ -1879,7 +1879,7 @@ def evaluate_property_stack(self):
                 typedef = self._parse_typedef( self.stack )
                 name = typedef['name']
                 klass = self.classes[ self.curClass ]
-                klass[ 'typedefs' ][ self.curAccessSpecifier ].append( name )
+                klass[ 'typedefs' ][ self.curAccessSpecifier ].append( typedef )
```

Comment:
@tpaviot I would like to, but unfortunately I cannot access the bitbucket website from my country, sorry for this, the fix on cppheaderparser is because for the internal typedef it only returns the name as a string, with the change it returns the whole typedef.